packages/form-js-playground/rollup.config.js

```diff
@@ -71,6 +74,17 @@ export default [
     plugins: pgl([
       resolve(),
       commonjs()
-    ])
+    ]),
+    onwarn
   }
-];
\ No newline at end of file
+];
+
+// TODO(@barmac): remove once https://github.com/moment/luxon/issues/193 is resolved
+function onwarn(warning, warn) {
```

Comment:
Sure that we want to ignore this? Can the built bundle still be consumed (conditions be evaluated)?

Reply:
==> Let's add an integration test for this in form-js.

Reply:
What would you expect to test? Something like https://github.com/bpmn-io/form-js/pull/403/commits/3c0647231ddad54f2a2ad53389ead50c976538e9  ?